docs/index.md

```diff
@@ -22,7 +22,9 @@ clicks away.
 
 ## What's New
 
-Follow [@ASFHyP3](https://twitter.com/ASFHyP3 "https://twitter.com/ASFHyP3" ){target=_blank} on Twitter, or check our [What's New](whats_new.md "HyP3 What's New" ){target=_blank} page to keep up to date on all things HyP3!
```

Comment:
Should we keep the link to our What's New page here?

Reply:
Yes, if we embed the mastodon timeline on the what's new page (suggestion below)